README.md

```diff
@@ -110,3 +110,50 @@ Future<dynamic> backgroundMessageHandler(Map<String, dynamic> message) async {
 
 ### iOS
 Make sure that you have a `NSPhotoLibraryUsageDescription` entry in your `Info.plist`.
+
+If you don't have any push notifications integrated in your application you need to:
+1. Enable push notification in you provisioning profile.
+2. Add Capability `Push Notifcations` in your mobile application.
```

Comment:
I'm not very familiar with iOS development, so I guess this is trivial, but we could maybe provide more detailed info about these 2 steps.

Reply:
I will remove 1. since this was legacy way. Right now it's done through XCode.